.github/workflows/ci.yml

```diff
@@ -40,14 +39,6 @@ jobs:
           kind_version: 'v0.11.1'
           kubernetes_version: '1.19.11'
           kind_image: 'kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729'
-        - ruby: '2.6.6'
```

Comment:
I don't we can support Ruby 2..6 after the Pysch 4 upgrade - there are incompatibilities with ActiveSupport.